apps/mull-ui/src/app/pages/login/login.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { CustomTextInput } from '@mull/ui-lib';
+import { Link } from 'react-router-dom';
+import { History } from 'history';
+import * as Yup from 'yup';
+import { useFormik } from 'formik';
+
+import logo from '../../../assets/mull-logo.png';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFacebookSquare, faGoogle, faTwitter } from '@fortawesome/free-brands-svg-icons';
+
+import './login.scss';
+
+export interface LoginProps {
+  history: History;
+}
+
+export const Login = ({ history }: LoginProps) => {
+  const formik = useFormik({
+    initialValues: {
+      email: '',
+      password: '',
+    },
+
+    validationSchema: Yup.object({
+      email: Yup.string().required('Email is required.').email('Email format is incorrect'),
+      password: Yup.string().required('Password is required.'),
+    }),
+
+    onSubmit: (values) => {},
+  });
+
+  return (
+    <form onSubmit={formik.handleSubmit}>
+      <div className="login-container">
+        <img src={logo} className="login-mull-logo" alt="Mull logo" />
+
+        <CustomTextInput
+          title="Email"
+          fieldName="email"
+          value={formik.values.email}
+          onChange={formik.handleChange}
+          hasErrors={formik.touched.email && !!formik.errors.email}
+          errorMessage={formik.errors.email}
+        />
+
+        <CustomTextInput
+          title="Password"
+          fieldName="password"
+          value={formik.values.password}
+          onChange={formik.handleChange}
+          hasErrors={formik.touched.password && !!formik.errors.password}
+          errorMessage={formik.errors.password}
+          password
+        />
+
+        <button type="submit" className="login">
+          Login
+        </button>
+
+        <div className="sign-up">
+          Don't have an account?{' '}
+          <Link to="/register" className="register-login-page">
+            {' '}
+            Sign up here!
+          </Link>
+        </div>
+
+        <div className="separator">or</div>
+
+        <button className="twitter">
+          <FontAwesomeIcon icon={faTwitter} size="2x" />
+          &nbsp; Continue with Twitter
```

Comment:
You can use {' '} instead of &nbsp; to be consistent
```suggestion
          {' '}Continue with Twitter
```
as for the buttons below

Reply:
Tried it, this doesn't seem to work. From the images below when using {' '} it actually doesn't create a space. 

![image](https://user-images.githubusercontent.com/35787279/99865047-2e20c680-2b75-11eb-8eff-9238512763a8.png)

![image](https://user-images.githubusercontent.com/35787279/99865052-37119800-2b75-11eb-88ee-8a1529708b38.png)

Reply:
That's my bad, I assumed it would work without testing

Reply:
Alternatively, you can put a margin between the icon and put the text in a div. Idk if using &nbsp; is a good idea.